apache/zeppelin
# [WIP][DOC] 0.7.0 release note

### What is this PR for?
Add 0.7.0 release note

### What type of PR is it?
Documentation

### Todos
* [x] Confirm release date and update list of people who verified release once vote pass.
* [x] Update doap.rdf

### How should this be tested?
Open [zeppelin-release-0.7.0.md](https://github.com/minahlee/zeppelin/blob/28dfd26fa6f6fe87cd629770674dcbba10b50e04/releases/zeppelin-release-0.7.0.md) and see if there is missing feature that needs to be mentioned in release note.

### Screenshots (if appropriate)
Download page
![screen shot 2017-02-04 at 10 48 07 pm](https://cloud.githubusercontent.com/assets/8503346/22618917/2790dda6-eb2c-11e6-9834-5612d03f42af.png)


### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? -
